commonui/src/main/java/com/hyperwallet/android/ui/common/view/error/DefaultErrorDialogFragmentPresenter.java

```diff
@@ -21,10 +21,8 @@
 import androidx.annotation.NonNull;
```

Comment:
This class and the interface being implemented can be removed. After the changes, this class is no longer necessary